docs/installation.rst

```diff
@@ -80,6 +93,15 @@ If on CentOS or RHEL, use this command:
 
    sudo yum install -y gcc make cmake bzip2-devel expat-devel file-devel glib2-devel libcurl-devel libmodulemd-devel libxml2-devel python36-devel rpm-devel openssl-devel sqlite-devel xz-devel zchunk-devel zlib-devel
 
+Ensure your virtual environment uses system wide packages
+*********************************************************
+
+.. code-block:: bash
+
+    grep "include-system-site-packages" pyvenv.cfg
```

Comment:
We should probably give users some indication of where to look for pyvenv.cfg.

Reply:
I put usual path, may we can indicate to use "which python" if in virtualenv yet.